doc/source/install/index.rst

```diff
@@ -0,0 +1,76 @@
+.. _install:
+
+=================================
+Installing Percona XtraDB Cluster 
+=================================
+
+|PXC| supports most 64-bit Linux distributions.
+Percona provides packages for popular DEB-based and RPM-based distributions:
+
+* Debian 6 ("squeeze")
+* Debian 7 ("wheezy")
+* Debian 8 ("jessie")
+* Ubuntu 12.04 LTS (Precise Pangolin)
+* Ubuntu 14.04 LTS (Trusty Tahr)
+* Ubuntu 15.04 (Vivid Vervet)
+* Ubuntu 15.10 (Wily Werewolf)
+* Red Hat Enterprise Linux / CentOS 5
+* Red Hat Enterprise Linux / CentOS 6
+* Red Hat Enterprise Linux / CentOS 7
+
```

Comment:
update the list of supported platforms, no Debian 6, Vivid and CentOS 5, Xenial is missing